ScoreSaber/Core/ReplaySystem/UI/DesktopMainImberPanelView.cs

```diff
@@ -153,25 +155,28 @@ public void UpClick() {
 
 
         private void RecursivelyDisableItalics(GameObject obj) {
+            TextMeshProUGUI[] textMeshProUGUIs = obj.GetComponentsInChildren<TextMeshProUGUI>();
             var textComponent = obj.GetComponent<TextMeshProUGUI>();
             if (textComponent != null) {
                 textComponent.fontStyle = FontStyles.Normal;
             }
```

Comment:
This is not needed. `GetComponentsInChildren` already returns this one as well. Refer to the docs:
https://docs.unity3d.com/ScriptReference/Component.GetComponentsInChildren.html

> Gets references to all components of type T on the same GameObject as the component specified, and any child of the GameObject.

Reply:
Ah, didnt know that. Thank you. Fixed